setup.cfg

```diff
@@ -39,6 +39,27 @@ wayland =
   # NOTE: When updating the major or minor version of pywlroots here, also update:
   # tox.ini, docs/manual/install/index.rst, docs/manual/wayland.rst
   pywlroots==0.16.4
+widgets =
+  dbus_next
+  imaplib2
+  iwlib
+  keyring
+  mailbox
+  psutil
+  pulsectl
+  pulsectl_asyncio
+  python-mpd2
+  pytz
+  xdg
+  xmltodict
+all =
+  # All user dependencies i.e. excluding dev deps.
+  %(ipython)s
+  %(wayland)s
+  %(widgets)s
+  bowler
+  dbus-next
```

Comment:
This is included in `widgets`

Reply:
Ah nice

Reply:
Ooh the widgets one has an underscore and the test one has a dash. I guess pip normalises those when installing things